### ISSUE TITLE
Question: Autostart Redshift on kUBUNTU 16.04

### ISSUE THREAD

**Issue by GITMichaelK** [jonls/redshift#567]
I'm a bit confused how to start Redshift: For my kUBUNTU 16.04 I installed redshift 1.10-5 and the Plasmaoid 1.0.18, where I have set the location and day / night values and the checkbox 'Autostart'. As everything is defined in the Plasmoid, I did not create a Configuration file nor any additional Autostart setting.

It's running and if I switch it manually OFF > ON or DefaultIcons OFF > ON in the Plasmaoid (lets say at 3 p.m.), brightness runs down (from approx. 4:30 p.m.) as desired. But without such clicks it stays with the "Day"-values all night.

How to run it automatically?

**Comment by breznak** [comment]
search for common "how to run an app automatically on linux", ie one solution would be adding redshift to your `~/.bashrc`

**Comment by GITMichaelK** [comment]
Thanks for your response, but I couldn't solve it: I know how to add programs to autostart on kUBUNTU and did it successfully for test with firefox and filezilla.
For redshift there are installed actually:
- redshift (version 1.10): /usr/bin/redshift
- plasmoid (version 1.0.18): ~/.local/share/plasma/plasmoids/org.kde.redshiftControl

Plasmoid having the AutoStart-option activated, which makes it starting switched ON, otherwise OFF.
Using `ps -e | grep redsh` I found, that if process 'redshift' is running, color-adjustment works fine, it doesn't work if that process is not running. I can start that process manually  from the plasmoid, switching it from OFF to ON.
What is the right file (incl. options?) to run from AutoStart? 
Actually I have there `Exec=/usr/bin/redshift` but this is not successful.

**Comment by breznak** [comment]
As you are running the plasmoid on KDE, I have the same setup (on Neon), and didn't have to do anything..the plasmoids autostart themselves. Can you try reinstalling redshift and the plasmoid, or search for specific kUbuntu issues?

**Comment by GITMichaelK** [comment]
Thanks for your reply and yout confirmation, that it works with the plasmaoid. First I tried my NEON (VirtualMachine) where it did not work, may be caused ba virtual environment. There I've seen the plasmoid in the control panel, so I went back to my normal kUBUNTU 16.04. and moved it from desktop to control panel. After restart, found the following differences:
- Redshift works now as designed and expected!
- Process 'redshift' `ps -e | grep red` is running now without any additional change done by myself.
Thanks for the help your encouragement: For me the issue is solved.
